openslides/core/static/js/core/base.js

```diff
@@ -294,6 +297,29 @@ angular.module('OpenSlidesApp.core', [
     }
 ])
 
+.factory('StartupExtraData', [
+    'Assignment',
```

Comment:
Please do not make 'Assignment' to be a dependecy of our core module. So solve this extraData thing we need another approach. Maybe something like we do for custom django settings. See https://github.com/OpenSlides/OpenSlides/blob/master/openslides/core/views.py#L181